frontend/src/App.js

```diff
@@ -1,4 +1,7 @@
+
 import React, { Component } from 'react';
+// import SignIn from './components/sign_in/sign_in';
+// import SignUp from './components/sign_up/sign_up';
```

Comment:
I'd love to avoid commented code where possible. It makes stuff harder to read/maintain, and if we need to add it back later, we can always pull it from git history.